### ISSUE TITLE
Plural of Epoch is Epochs.  Linter wants Epoches

### ISSUE THREAD

**Issue by zeroflucs-adam-bishop** [golangci/golangci-lint#6080]
### Welcome

- [x] Yes, I'm using a binary release within 2 latest [releases](https://github.com/golangci/golangci-lint/releases). Only such installations are supported.
- [x] Yes, I've searched similar [issues on GitHub](https://github.com/golangci/golangci-lint/issues) and didn't find any.
- [x] Yes, I've read the `typecheck` section of the [FAQ](https://golangci-lint.run/docs/welcome/faq/#why-do-you-have-typecheck-errors).
- [x] Yes, I've tried with the standalone [linter](https://golangci-lint.run/docs/linters/) if available (e.g., gocritic, go vet, etc.).
- [x] I agree to follow this project's [Code of Conduct](https://github.com/golangci/golangci-lint?tab=coc-ov-file)

### How did you install golangci-lint?

Brew

### Description of the problem

Linter complains that Epochs is not plural and wants Epoches.  Most english words ending in ch pluralize with -es but not epoch.

### Version of golangci-lint

2.4.0

### Configuration

N/A

### Go environment

1.25.0

### Verbose output of running

N/A

### A minimal reproducible example or link to a public repository

<details>

```go
var epochs
```

</details>

### Validation

- [x] Yes, I've included all information above (version, config, etc.).

### Supporter

- [ ] I am a sponsor/backer through [GitHub](https://github.com/sponsors/golangci/) or [OpenCollective](https://opencollective.com/golangci-lint)

**Comment by zeroflucs-adam-bishop** [comment]
AI Overview
For words ending in '-ch', most nouns add '-es' (e.g., church → churches) to form the plural by adding an extra syllable. However, a key exception is for words where the '-ch' sounds like a hard 'k' (e.g., monarch, stomach), which are pluralized by adding only '-s'

**Comment by ldez** [comment]
Hello,

| Section                                                       | Status |
|---------------------------------------------------------------|--------|
| Welcome                                                       | 🟠 |
| Description of the problem                                    | 🟠 |
| Version of golangci-lint                                      | 🟠 |
| Configuration                                                 | 🔴 |
| Go environment                                                | 🔴 |
| Verbose output of running                                     | 🔴 |
| A minimal reproducible example or link to a public repository | 🔴 |
| Validation                                                    | 🔴 |
| Supporter                                                     | 🟢 |

Legend:
- 🟢: The section is complete and valid.
- 🟠: The section is incomplete.
- 🔴: The section is missing or invalid.

**Comment by zeroflucs-adam-bishop** [comment]
Yep sorry, was a different linter causing this.

**Comment by ldez** [comment]
Please consider donating or asking your company to do so.

This will be appreciated, thank you :heart:

https://donate.golangci.org/ 
https://donate.ldez.dev/
